elements/lisk-p2p/src/utils/misc.ts

```diff
@@ -152,6 +152,40 @@ export const getUniquePeersbyIp = (
 	return [...peerMap.values()];
 };
 
+export const removeCommonIPsFromLists = (
+	peerList: ReadonlyArray<P2PPeerInfo>,
+	controlPeerList: ReadonlyArray<P2PPeerInfo>,
+): ReadonlyArray<P2PPeerInfo> => {
+	const peerMap = new Map<string, P2PPeerInfo>();
+
+	for (const controlPeer of controlPeerList) {
```

Comment:
I can not think of a better way to improve this code for now, but let's catch up and see if we can make it better.